setup.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-""" 
+"""
 resolvedvelocities calculates the 2D plasma drift velocity
```

Comment:
2D or 3D?

Reply:
We say 3D other places, so we can use that.  It's kind of wierd because the plasma drift velocity is kind of assumed to be 2D (perp to B).

Reply:
We say 3D other places, so we can use that.  It's kind of wierd because the plasma drift velocity is kind of assumed to be 2D (perp to B).
```suggestion
resolvedvelocities calculates the 3D plasma drift velocity
```